oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexStatistics.java

```diff
@@ -26,13 +27,13 @@
 
 import co.elastic.clients.elasticsearch._types.query_dsl.Query;
 
+import org.apache.jackrabbit.guava.common.base.Ticker;
```

Comment:
Maybe move back to the other guava imports?

Reply:
Eventually, these will all go. The other ones are misplaced because they were automatically renamed.